Src/FluidCaching/LifespanManager.cs

```diff
@@ -7,6 +7,8 @@ namespace FluidCaching
 {
     internal class LifespanManager<T> : IEnumerable<INode<T>> where T : class
     {
+        private const int MaxItemsInBag = 20;
```

Comment:
`20` is not max item count in a bag. The max item count in a bag is calculated as `Math.Max(capacity / 20, 1)`. `20` is something like the number of bags which should be approximately enough to store the requested capacity of items. So that one bag cannot store more than 5% of capacity.